dropbear_install

```diff
@@ -60,16 +78,14 @@ build ()
   fi
 
   # if TMPDIR is set leave it alone otherwise set
-  [ -z $TMPDIR ] && TMPDIR='/tmp/dropbear_initrd_encrypt'
+  [ -z "$TMPDIR" ] && TMPDIR='/tmp/dropbear_initrd_encrypt'
 
   # check if TMPDIR exsists if not make it
-  [ -d $TMPDIR ] || mkdir -p $TMPDIR
-
-  umask 0022
+  [ -d "$TMPDIR" ] || mkdir -p "$TMPDIR"
 
   [ -d /etc/dropbear ] && mkdir -p /etc/dropbear
 
-  copy_openssh_keys || generate_keys
+  use_dropbear_keys || copy_openssh_keys || generate_keys
```

Comment:
Did you get your Git branches cross-wired? I thought this stuff was in a different PR.